### ISSUE TITLE
Everything Is All It Takes: A Multipronged Strategy for Zero-Shot Cross-Lingual Information Extraction

### ISSUE THREAD

**Issue by pocca2048** [pocca2048/ML-paper-reading#18]
**Conference** : EMNLP 2021
**Link** : https://arxiv.org/pdf/2109.06798.pdf
**Authors' Affiliation** : Johns Hopkins University
**TL;DR** : Data projection + self-training을 시험해봤다. 어느 한 가지 방법이 모든 것에서 잘하기 않기 때문에 zero-shot training에서 잘하려면 다 해봐야 한다.

**Summary** :
참고) [POS tagging vs Dependency parsing vs Constituency parsing](https://www.analyticsvidhya.com/blog/2020/07/part-of-speechpos-tagging-dependency-parsing-and-constituency-parsing-in-nlp/)

## method
* augmenting the source language training data with data in the target language—either via projection of the source data to the target language (so-called “silver” data)
* via self-training with translated text

* Data projection = src 언어에 있는 word-level annotations을 tgt 언어로 word-to-word alignments를 통해 transfer하는 것.
  * 먼저 src -> tgt 으로 MT를 사용해서 번역
  * word alignment tool을 사용해서 alignment를 얻음 (fast-align, Awesome-align)
  * alignment를 사용해서 annotation을 project

레이블이 word-level인 것들은 sub-words에서 첫 번째를 제외하고 마스크 시킴.

## 6 Cross-lingual Transfer Results
One might optimistically consider that the latest multilingual encoder (in this case XLM-R) in the zero-shot setting would achieve the best possible performance. However, in our extensive experiments in Table 4 and Table 5, we find that the zero shot approach can usually be improved with data projection.

## 8 Conclusions
* adding projected silver training data overall yields improvements over zero-shot learning
* the best setup is usually task dependent.
* In multilingual experiments, we find that silver data tends to help languages with the weakest zeroshot performance, and that it is best used separately for each desired language pair instead of in joint multilingual training.
* self-training helps.
